ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h

```diff
@@ -4,10 +4,13 @@
 //
 
 #import <Foundation/Foundation.h>
-#import <Purchases/Purchases.h>
+@import RevenueCat;
 
 NS_ASSUME_NONNULL_BEGIN
 
+// JOSH: This is here because RCDangerousSettings is not in v4
+@interface RCDangerousSettings: NSObject
+@end
```

Comment:
This needs to get added to v4 (It was added in v3 in https://github.com/RevenueCat/purchases-ios/pull/1219)